applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -145,6 +152,7 @@ func genDNSHostnames(workerGroupSpec ray.WorkerGroupSpec, replicaIndex int) (str
 
 // inject subdomain and TPU_WORKER_HOSTNAMES into pods for TPU multi-host initialization
 func injectHostnames(hostNames string, envPath string, container corev1.Container, patches *[]patch) {
+	klog.Infof("Injecting TPU_WORKER_HOSTNAMES to Pod: %s", hostNames)
```

Comment:
Not sure if these are really needed. Consider making these verbose logs instead.

Reply:
Done, I changed some of the longer logging statements to verbosity level 1 and added a new Makefile command to deploy the webhook with verbose logging.